### ISSUE TITLE
Resurrect the Food API and implement the endpoints

### ISSUE THREAD

**Issue by qasim** [cobalt-uoft/cobalt#22]
https://github.com/cobalt-uoft/cobalt/tree/master/api/food is where the food API is located. It was abandoned.

It doesn't have a scraper, but map.utoronto.ca is your best friend to implement one at https://github.com/cobalt-uoft/uoft-scrapers/issues/15.

**Comment by kashav** [comment]
[Scrapers](https://github.com/cobalt-uoft/uoft-scrapers/tree/master/uoftscrapers/scrapers/food) are ready to be used.

I've also started work on the endpoints [here](https://github.com/kshvmdn/cobalt/tree/master/src/api/food). Datasets are currently not available for the food json so I'm using [this](https://github.com/cobalt-uoft/cobalt/files/201133/food.json.zip). Import the collection with:

```
mongoimport --db cobalt --collection foods --file ./path/to/food.json 
```

**Comment by qasim** [comment]
Sweet! We will ship 0.4.0 after cobalt-uoft/uoft-scrapers@0.2.0 is shipped, at that point the scrapers will be active for food in the MongoDB collection called `food` (no `s`).

**Comment by kashav** [comment]
I was originally using `food`, but whenever I tried accessing the endpoint, it'd create a new collection named `foods`.

I think it's something that Mongoose does automatically.

**Comment by kashav** [comment]
It looks like we can override that by setting our own name:

```
new Schema({..}, { collection: 'food' })
```

http://stackoverflow.com/questions/10547118/why-does-mongoose-always-add-an-s-to-the-end-of-my-collection-name

**Comment by qasim** [comment]
Hmmmmmm.... maybe we should have it as `foods` then, just to stay consistent. My bad.

**Comment by kashav** [comment]
The food [endpoints](https://github.com/kshvmdn/cobalt/tree/master/src/api/food) are nearly complete, including `filter` (this one may need to be revised). The initial [documentation](https://github.com/kshvmdn/documentation/tree/master/endpoints/food) is done as well. 

I also had to change the scraper a bit so that `tags` were all lowercase (i'll make a PR when it's finalized). Here's the updated [food data](https://github.com/cobalt-uoft/cobalt/files/201702/food.json.zip) for testing.

**Comment by qasim** [comment]
Submit a PR when you're ready, also a few examples of how to use the filter endpoint. Then we can go over if it's ideal or can be tweaked. Everything's looking good! I'll be busy this week so 0.4.0 probably won't be out the door for another few days (since I still have to finish up that textbook scraper).

**Comment by qasim** [comment]
Merged #47. Once you've made a PR for the documentation, I will merge it when 0.4.0 ships.
